source/contracts/reporting/ReportingToken.sol

```diff
@@ -26,7 +26,7 @@ contract ReportingToken is DelegationTarget, Typed, Initializable, VariableSuppl
         require(_market.getNumberOfOutcomes() == _payoutNumerators.length);
         market = _market;
         payoutNumerators = _payoutNumerators;
-        // TODO: call a function on `self.getUniverse()` that logs the creation of this token with an index for the market, function needs to verify that caller is `universe.isContainerForReportingToken(thisToken)`
+        getUniverse().logCreateReportingToken(this, market);
```

Comment:
Recommend against implementing any logs until we know what the UI needs.  Also, anywhere you see a "need to log this" comment feel free to delete.